tests/serverpod_test_server/generated/openapi/.redocly.lint-ignore.yaml

```diff
@@ -0,0 +1,574 @@
+# This file instructs Redocly's linter to ignore the rules contained for specific parts of your API.
+# See https://redoc.ly/docs/cli/ for more information.
+openapi.json:
```

Comment:
Question: why is this file needed? If we want this linter then I can we not simply follow the linting rules instead of opting out of them?

Reply:
Ok. removed.

Reply:
If this file is still here, this shouldn't be marked as resolved, right?

It's always much better to comply with linting rules, rather than ignoring them unless there is a _very_ good reason for it. If a linting rule is ignored, it should be documented why this is the case.

Reply:
I thought the tags description, post operation summary, and unused components could be ignored. However, the Redocly CLI has an option to generate ignore rules, so I will remove that file.